include/picongpu/param/physicalConstants.param

```diff
@@ -46,18 +46,22 @@ namespace picongpu
 
         // Electron properties
         /** unit: kg */
-        constexpr float_64 ELECTRON_MASS_SI = 9.109382e-31;
+        constexpr float_64 ELECTRON_MASS_SI = 9.1093837015e-31;
```

Comment:
```suggestion
        constexpr float_64 ELECTRON_MASS_SI = 9.1093837139e-31;
```
from [wikipedia](https://en.wikipedia.org/wiki/Electron_mass).

Reply:
Okay. Also found [this](https://en.wikipedia.org/wiki/Atomic_units) which has the value you used. One of them must be lying... -.-